.circleci/config.yml

```diff
@@ -227,8 +289,18 @@ workflows:
           requires:
             - deci-common/persist_version_info
             - deci-common/codeartifact_login
+      - build_and_publish_beta:
```

Comment:
@shaydeci  do we want to build and publish the container at every push? not only on release?

Reply:
only on release